locksmith/src/controllers/v2/checkoutController.ts

```diff
@@ -20,30 +19,19 @@ export const createOrUpdateCheckoutConfig: RequestHandler = async (
 ) => {
   const id: string | undefined = request.params.id
   const { config, name } = request.body
-  const userAddress = request.user!.walletAddress
 
   if (!(config && name)) {
     return response.status(400).send({
       message: 'Missing config or name',
     })
   }
 
-  if (id) {
-    const authorized = await isAuthorizedToManageConfig(userAddress, id)
-    if (!authorized) {
-      return response.status(403).send({
-        message:
-          'Unauthorized: You must be a Lock Manager of at least one associated lock or the creator.',
-      })
-    }
-  }
-
```

Comment:
I think that was good though. We should make it so that any manager on any lock in the config can update the config.

Reply:
okay okay got it!